io.podman_desktop.PodmanDesktop.yml

```diff
@@ -0,0 +1,95 @@
+app-id: io.podman_desktop.PodmanDesktop
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: run.sh
+separate-locales: false
+finish-args:
+  - "--socket=session-bus"
+  - "--socket=wayland"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--device=dri"
+  - "--filesystem=home"
+  - "--filesystem=xdg-run/podman"
+  - "--share=network"
+  - "--talk-name=org.freedesktop.Notifications"
+build-options:
+  append-path: /usr/lib/sdk/node16/bin
+modules:
+  # needs yarn to build the project
+  - name: yarn
```

Comment:
Yarn is available in the node sdk extension, you don't need to bundle it.

Reply:
Ok 👍

Reply:
fixed